### PR TITLE
Bpt price feed

### PR DESCRIPTION
**Pull request by MrDeadCe11**
updated chainlink oracle consumer.  added a method to get an array of token values.

### CODE REVIEW

**Comment by JulissaDantes** [comment]
Why is the contract called `PriceFeedInterface` and not the filename?

**Review by JulissaDantes — DISMISSED** [review]
This branch shouldnt be merged until we have the tests for the new contract

**Comment by JulissaDantes** [comment]
Why do we have getLatestPrice and getTimeStamp? I suppossed you intended it to work as you implemeneted it on the fetch contract, so Im going to update the contract to take the address. @MrDeadCe11

File: package.json [hunk 1]
```diff
@@ -71,5 +71,8 @@
   },
   "eslintConfig": {
     "extends": "@enzymefinance/eslint-config"
+  },
+  "dependencies": {
```
Comment (jrhite) [inline package.json:75]: can we add this to `devDependencies` instead of `dependencies` instead?

**Pull request merged**